dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart

```diff
@@ -30,16 +30,30 @@ class _MainWidgetState extends State<_MainWidget> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(title: const Text('CheckBoxListTile')),
-      body: Center(
-        child: CheckboxListTile(
-          value: _checked,
-          onChanged: (bool? value) {
-            setState(() {
-              _checked = value!;
-            });
-          },
-          title: const Text('a check box list title'),
-        ),
+      body: ListView(
+        children: <Widget>[
+          CheckboxListTile(
+            autofocus: true,
```

Comment:
Do you have an open issue for fixing the focus? if so we should add a todo here.

Reply:
Left a TODO in `main.dart`, since I'm using `autofocus` a lot in the app.